README.md

```diff
@@ -170,6 +172,32 @@ Can you think of more? Let's discuss in the issues section!
   ```
 </details>
 
+#### InstantExecutorExtension
+
+Support for `InstantTaskExecutorRule` for JUnit 5. 
+
+<details open>
+  <summary>Kotlin</summary>
+
+  ```kotlin
+  dependencies {
+      testImplementation("io.github.neboskreb:instant-task-executor-extension:1.0.0")
```

Comment:
Thank you for adding blocks for both Kotlin and Groovy here! As for the artifact itself, I would prefer to declare it as a variable in the template file and have it resolved automatically when the readme is generated. This makes it a bit easier to maintain and upgrade if the version ever increments.

To do this, open `build-logic/src/main/kotlin/Dependencies.kt` and add a declaration for these Maven coordinates to it (the full thing, including group, artifact and version), e.g. `const val instantTaskExecutorExtension = ...`. Then, in the template file, change these lines to:

```kotlin
testImplementation("${libs.instantTaskExecutorExtension}")
```

Reply:
Fixed